modules/ROOT/partials/menu-item-ids/core-menu-items.adoc

```diff
@@ -3,15 +3,15 @@
 |Menu item identifier |xref:menus-configuration-options.adoc#examplethetinymcedefaultmenuitems[Default Menu Location] |Description
 |`+align+` |Format |Changes alignment to the current block or selection.
 |`+backcolor+` |Format |Applies background color to selection.
-|`+blockformats+` |Format |Applies block formats to current selection.
+|`+blocks+` |Format |Applies block formats to current selection.
 |`+bold+` |Format |Applies bold format to current selection.
 |`+codeformat+` |Format |Applies inline code format to current selection.
 |`+copy+` |Edit |Copies the current selection into clipboard.
 |`+cut+` |Edit |Cuts the current selection into clipboard.
 |`+forecolor+` |Format |Applies foreground/text color to selection.
 |`+formats+` |Format |Menu of all available formats.
```

Comment:
This was also renamed to `styles` I believe? Will need to be renamed and moved, to keep alphabetical order. And this description isn't great... We could steal the description from the toolbars list? `Dropdown list with styles to apply to selection.`